apps/cow-fi/components/Chart/index.tsx

```
@@ -79,6 +81,7 @@ export function formatDelta(delta: number | null | undefined) {
 }
 
 export function Chart({ prices, height, width, timePeriod }: ChartProps) {
+  const theme = useTheme() as CowProtocolTheme
```

ESLint

[error] 'theme' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)